versioned_docs/version-3.0.0-beta/organization/personal-accounts.md

```diff
@@ -0,0 +1,28 @@
+---
+id: personal accounts
+title: Personal Accounts
+sidebar_label: Personal Accounts
+slug: /organization/personal-accounts
+---
+
+## Personal Accounts
```

Comment:
I think this should be a completely separate section from the organization. It doesn't belong to the organization at all. 

This section should contain 2 pages for now - 
- Overview
- Login Connection

Later we will be API token in this section as well when the feature is built

Reply:
Added to the new section

Reply:
I don't see those changes, also rename the files to
- `Overview.md` -> `overview.md`
- `Login-Connection.md` -> `login-connection.md`

Reply:
Also I don't see the overview page at all, something is wrong, please check